### ISSUE TITLE
Guidelines about when placeholders should be generics (`SOURCE`) v/ real-looking items (`/path/to/source.file`)

### ISSUE THREAD

**Issue by sknorr** [SUSE/doc-styleguide#174]
Recently, during a review, I noticed that placeholders were using real-looking items (like `/path/to/source.file`) rather than the generic terms (`SOURCE`) I expected. Looking at the style guide, I saw no rule about this. We do use generic placeholders in most cases already where they're related to user input, i.e. in `<replaceable/>` elements (afaik).

Major exceptions to this imo fall along a few lines making them different:

* When the placeholder is related to output only (like user names in prompts are)
* When it is meant explicitly as an example rather than a placeholder (like user names most everywhere else)
* Our networking entities are a clear case where the lines are blurred. :)

**Comment by lproven** [comment]
I can see problems with `sentences/split/up/like/that`... e.g. in languages where it requires Unicode or something. `detta/är/dåligt/exempel`. Guidance against it seems reasonable to me.

**Comment by sknorr** [comment]
This is true, though currently not too relevant--Padraig mentioned that replaceables are treated as untranslatable (at least currently).

I should note, `/path/to/source.file` is strictly an example--any number of other items may also be in replaceables, e.g.:

* `SHA_HASH` v/ `5848458`
* `BOOLEAN` v/ `true/false`
* `ITEM_NAME` v/ `my-item-1`
* `USER_NAME` v/ `tux`

**Comment by dariavladykina** [comment]
Currently in `doc-sle`, there are 84 suspicious instances of a seemingly wrong use of replaceables, please see the list.

**Comment by cwickert** [comment]
@dariavladykina, how did you compile this list? Search for `replaceable`s that are not capitalized?
 
I think that if we stick to generic placeholders, we should (continue to) capitalize them. IHMO it makes the placeholders more obvious. But the main question is about generic vs. real looking placeholders.

**Comment by dariavladykina** [comment]
Closing this for now until when we find more examples of wrong placeholder usage.

**Comment by cwickert** [comment]
Agreed. Nevertheless we should try to fix the ones from your list.
